src/lib/y2users/password_helper.rb

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2users"
+require "y2users/users_simple"
+
+Yast.import "UI"
+Yast.import "Popup"
+Yast.import "Report"
+
+module Y2Users
+  # Mixin to hold shared helper methods related to the user password
```

Comment:
Please, document that this helper is used by dialogs, because it takes cares of UI things like setting the focus, etc.